README.md

```diff
@@ -165,22 +152,17 @@ spec:
     links:
       - title: Repository
         url: ${{ steps.publish.output.remoteUrl }}
-      - title: Open in catalog
-        icon: catalog
-        entityRef: ${{ steps.register.output.entityRef }}
 ```
 
 You can also visit the `/create/actions` route in your Backstage application to find out more about the parameters this action accepts when it's installed to configure how you like.
 
 ### Environment setup
 
-The environment needs to have either `copier` installed and be available in the `PATH` or access to a `docker` daemon so it can spin up a docker container with `copier` available.
-
-If you are running Backstage from a Docker container and you want to avoid calling a container inside a container, you can set up `copier` in your own image, this will use the local installation instead.
```

Comment:
Probably want to keep this too

Reply:
but we're no longer allowing them to spin up a *new* container. They don't have an option.

Reply:
The first bit talks about having it in your path (i.e. for a local installation), this is about running it in Docker - which is effectively the same as running it locally. Probably isn't necessary to be fair, because folks should know that the local instructions hold for inside a container